openslides_backend/action/actions/user/set_password_self.py

```diff
@@ -39,3 +39,23 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         instance["id"] = self.user_id
         instance["password"] = self.auth.hash(new_pw)
         return instance
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        if self.auth.is_anonymous(self.user_id):
+            raise ActionException("Can't set password for anonymous")
+
+        meeting_id = self.datastore.get(
+            FullQualifiedId(Collection("user"), self.user_id), ["meeting_id"]
+        ).get("meeting_id")
+
+        if meeting_id:
```

Comment:
`meeting_id` must always be set. Either remove the `if` or throw an error otherwise

Reply:
Ok, forget it - I was wrong.